nbs/030_session.ipynb

```diff
@@ -54,9 +54,9 @@
     "\n",
```

Comment:
<p>something happened with the rendering of outputs in nbreview here as well. Maybe its an nbdev_trust issue?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/spannerlib/pull/159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>